cub/cub/agent/agent_batch_memcpy.cuh

```diff
@@ -802,7 +802,8 @@ private:
       if (blev_buffer_offset < num_blev_buffers)
       {
         BlockBufferOffsetT tile_buffer_id = buffers_by_size_class[blev_buffer_offset].buffer_id;
-        block_offset[i] = CUB_QUOTIENT_CEILING(tile_buffer_sizes[tile_buffer_id], BLOCK_LEVEL_TILE_SIZE);
+        block_offset[i] =
+          ::cuda::ceil_div(static_cast<BlockBufferOffsetT>(tile_buffer_sizes[tile_buffer_id]), BLOCK_LEVEL_TILE_SIZE);
```

Comment:
The cast here looks odd. I'll replace it by a unary `+` instead. It's purpose is to decay a proxy ref to its value_type.

Reply:
Yep, now performance is equal. But it's concerning that the unit tests didn't catch this.

Reply:
Thanks. I'll look into what's going wrong and why tests didn't catch it.